src/Features/Core/Portable/Completion/CompletionItem.cs

```diff
@@ -188,5 +189,24 @@ public override string ToString()
         {
             return DisplayText;
         }
+
+        private List<string> tags = null;
+        public IEnumerable<string> Tags
```

Comment:
I'm not a huge fan of this. I think we need to reduce the concept count on CompletionItem rather than add more goo.